example/example.go

```diff
@@ -30,14 +30,21 @@ import (
 	"mosn.io/proxy-wasm-go-host/wasmer"
 )
 
-var contextIDGenerator int32
-var rootContextID int32
+//go:embed data/http.wasm
+var exampleWasm []byte
```

Comment:
should not use //go:embed, otherwise we cannot build once and run multiple times with different wasm files

Reply:
fine with me. in wazero we use embed for our examples when the paths are hard-coded as is in this case